bool roundabout
bool crossing
uint8 node_id